src/cases/components/allocate-role/allocate-role.component.scss

```diff
@@ -0,0 +1,31 @@
+
+.govuk-back-link {
```

Comment:
Do you need to redefine this class. Is this not part of the GDS  global styles

Reply:
It was and actually the styles were referred to on the prototype, however I'm not sure why in particular but the prototype looked different to the class this refers to on the application. I had to shift the underline slightly and replace the arrow as it looked completely different. If you want me to redefine classes then I can do that but I will have to do a bit more editing on the link itself - redoing all styling

Reply:
Just look at the prototype you're using the correct class. However our class styling is significantly different from theirs. Are we using an older version of the GDS stylesheet.